clarity/src/vm/types/serialization.rs

```diff
@@ -612,7 +614,7 @@ impl Value {
                 }
 
                 if let Some(list_type) = list_type {
-                    Value::list_with_type(items, list_type.clone())
+                    Value::list_with_type(&StacksEpochId::latest(), items, list_type.clone())
```

Comment:
I think we'll want an explicit epoch ID here.  When we're working on Stacks 3.0, we'll want the serialization code to behave the same until it's strictly necessary to change it (especially if 3.0 introduces different behavior).